Ananya_Jain/landingPage/app.js

```diff
@@ -0,0 +1,123 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const nodemailer = require('nodemailer');
+const mongoose = require('mongoose');
+const ChannelModel = require('./models/channel')
+const dbUrl = "mongodb+srv://ananyajain:ananyaJAIN123@cluster0.j0todxb.mongodb.net/myDataBase?retryWrites=true&w=majority"
+
+
+// mongo
+const connectionParams = {
+  useNewUrlParser: true,
+  useUnifiedTopology: true
+}
+
+mongoose
+  .connect(dbUrl, connectionParams)
+  .then(()=>{
+    console.info("connected")
+})
+  .catch((e) => {
+    console.log("error:",e);
+  });
+
+var path = require('path');
+
+const app = express();
+const port = 3000;
+
+
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(express.static(path.join(__dirname, 'public')));
+
+
+// form
+app.get("/contact", function(req,res){
+  console.log("get");
+  res.sendFile("C:/Users/HP/Documents/Ananya/landing-page/landingPage/contact.html");
```

Comment:
Not the way the file will work Ananya! Do you think we will be able to see this file?

Reply:
I will correct this.